labs/StarterKit.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: HL Starter Kit
+parent: Labs
+---
+
+# Lab Name
+_HL Starter Kit_
+
+# Short Description
+
+The StarterKit provides a single window concept for all hyperledger projects and associated tools. The Starter kit reduces the barrier for entry by making the initial experience to research and developing project solutions painless. The tool makes it easier for the learners by covering end to end  with sample use cases and custom generated configuration by setup proof of concept for learning, experimenting and testing.
+
+
+# Scope of Lab
+
+The Starter Kit for Hyperledger Learners is the first step in developing solutions using the tools in the Hyperledger Greenhouse
+
+The Concept will cover technical and non-technical personalities by doing prototype from the project samples without using text editors. The project scoped to cover end-to-end custom generated configurations and ease their deployments. Learners will benefit with all possible use cases to launch and prototyping the blockchain network.
+
+For example, A fabric network was launched, followed by explorer and caliper will show visualized pitch to learners.
+This tool will provide a complete end to end scripts to launch the network from the selected projects and tools. The learners will have a sense of hyperledger platform and tools, which attract more developer communities and business to adopt.
+
+For an MVP, a certain level of use cases the customization has doable on Hyperledger fabric, explorer and can be deployed on single-node, docker-swarm, Kubernetes infra environments. However, this has to be expanded to other projects and tools which is in progress for besu,caliper,sawtooth...
+
+The tool allows to get experience with capabilities of :
+ 1, Customized Names for Domain, ORG’s, Orderer’s, Peer’s and CA’s
+ 2, Unlimited ORG’s, Orderer’s, Peers, CA’s  and channel  counts
+ 3, Toggle between the Orderer type : Solo  / RAFT
+ 4, Deployment between - SingleHost / Multihost Docker Swarm / Kubernetes setup
+ 5, Choices to select Chaincodes
+
+
+# Initial Committers
+
+- https://github.com/ravinayag
+- Name - Ravi Vasagam | ravinayag@gmail.com | Active Member - HL-LMDWG , HL-India Chapter 
+
+
+
+# Sponsor
+
+- Name - Bobbi Muscara (bobbi@ledgeracadamy.com) - Chair of the HL-LMDWG workgroup
+
+# Pre-existing repository
+
+- https://github.com/ravinayag/HL-StartertKit.git
```

Comment:
Looks like [one of the commits](https://github.com/ravinayag/HL-StartertKit/commit/82b3627d51eacb2ec1ec702ac51322609d8df82f) in this repo has not been signed off. Can you correct?

Reply:
Updated with signoff's